build.gradle

```diff
@@ -44,6 +45,24 @@ allprojects {
     }
 }
 
+apiValidation {
+
+    subprojects.forEach {
```

Comment:
subproject  contains only the modules declared in `settings.gradle` right? Otherwise you risk to dump also example and convections-plugin. Also a bit out of context, but looking at kaluga directories I noticed there is this [fields](https://github.com/splendo/kaluga/tree/develop/fields/src/commonMain/kotlin) path in the root but not sure what it is

Reply:
fields is removed in another branch already (accidental checkin by me from long ago), indeed these API dumps would help detect that stuff too. 

indeed only modules are in `subprojects`, I did not have to manually ignore or gitignore any `.api` files